pkg/option/config.go

```diff
@@ -1200,6 +1205,13 @@ func (c *DaemonConfig) Populate() {
 		c.LogOpt = m
 	}
 
+	if viper.IsSet(ConntrackGarbageCollectorIntervalDeprecated) {
+		val := time.Duration(viper.GetInt(ConntrackGarbageCollectorIntervalDeprecated))
+		c.ConntrackGCInterval = val * time.Second
```

Comment:
Consider adding a warning log that the old option is replaced by the new option?

Reply:
@joestringer `flags.MarkDeprecated` should do that automagically